src/main/scala/com/scalaSparkSkillValidation/models/Account.scala

```diff
@@ -0,0 +1,7 @@
+package com.scalaSparkSkillValidation.models
+import java.sql._
+
```

Comment:
Can you please add here the ScalaDoc, it's quite clear what we do here, but just to check how U do it

Reply:
Sure, I'll implement ScalaDoc into the project models

Reply:
ScalaDoc implemented